codechain/keystore/KeyType.py

```diff
@@ -0,0 +1,18 @@
+from enum import Enum
+
+
+class KeyType(Enum):
+    PLATFORM = 0
```

Comment:
You can use string values instead of numeric values here, or `auto` to 
```python
class KeyType(Enum):
	PLATFORM = "platform"

def get_table_name(key_type: KeyType):
    return key_type.value
```

Reply:
Thanks, I changed it.